components/card/card.css

```diff
@@ -0,0 +1,149 @@
+.card-wrapper {
+  display: flex;
+  flex-direction: column;
+  max-width: 350px;
+  margin: 1rem;
+  height: max-content;
+  border-radius: var(--border-radius);
+  background-color: var(--bg-color);
+}
+
+.basic-card .card-dismiss,
+.basic-card .card-badge {
+  display: none;
+}
+
+.card-img {
+  border-radius: var(--border-radius);
+  width: 100%;
+  height: auto;
+}
+
+.card-heading {
+  font-size: 1.1rem;
```

Comment:
you can use variables for figures so that reuse it.

Reply:
Yes, I'll make this change to all the files at the end. Thank you!